Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/util/GetHashMac.java

```diff
@@ -50,22 +57,21 @@ public static boolean IsValidHashMacFormat(String hashMac) {
 
         Pattern hashmac_pattern = Pattern.compile("[0-9a-f]{64}");
         Matcher matcher = hashmac_pattern.matcher(hashMac);
-        return matcher.matches();
+        return matcher.matches() && !(StringUtils.containsIgnoreCase(System.getProperty("os.name"), "mac") &&
```

Comment:
add a new isMac utility method in a new util class or an existing proper util class.

Reply:
Added to `azurecommon/util/Utils`